Lib/profiling/sampling/sample.py

```diff
@@ -632,6 +636,9 @@ def sample(
         case "flamegraph":
             collector = FlamegraphCollector(skip_idle=skip_idle)
             filename = filename or f"flamegraph.{pid}.html"
+        case "heatmap":
+            collector = HeatmapCollector(skip_idle=skip_idle)
+            filename = filename or f"heatmap_{pid}"
```

Comment:
I wonder if folks will want to have this saved by timestamp to compare historical runs?